system/Database/BaseBuilder.php

```diff
@@ -261,23 +252,22 @@ class BaseBuilder
     /**
      * Constructor
      *
-     * @param array|string $tableName
+     * @param array|string|null $from tablename or tablenames with or without aliases
+     *
+     * Examples of $tableName: `mytable`, `jobs j`, `jobs j, users u`, `['jobs j','users u']`
      *
      * @throws DatabaseException
      */
-    public function __construct($tableName, ConnectionInterface &$db, ?array $options = null)
+    public function __construct($from, ConnectionInterface &$db, ?array $options = null)
```

Comment:
Do I need to pass the connection interface by reference? Should the query builder be able to modify the database connection? I think not.

Reply:
Good point. I think all `&` with DB connection is not needed.
Because `$db` is an object. 
https://www.php.net/manual/en/language.oop5.references.php